mscore/musescore.cpp

```diff
@@ -2967,6 +2967,12 @@ AboutBoxDialog::AboutBoxDialog()
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
       copyRevisionButton->setIcon(*icons[int(Icons::copy_ICON)]);
+
+      copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
```

Comment:
I may have found a method to get rid of the <br>, actually 2 methods, a) via css `whit-space:pre-wrap` or for `.replace("\n", "<br/>")`, which would you prefer?